### ISSUE TITLE
Transfer from pytorch_transformer to transformer

### ISSUE THREAD

**Issue by Xiang-Pan** [songyouwei/ABSA-PyTorch#145]
I tried transfer from pytorch_transformer to transformer, the accuracy decrease a lot. 

What's the problem, and could the code group consider transfer to new APIS?

Thanks!

**Comment by fathouse** [comment]
I also had this problem, the average accuracy was reduced by more than 10%

**Comment by songyouwei** [comment]
That's because the specific order of some models keywords inputs (attention_mask, token_type_ids...) has been changed.
https://github.com/huggingface/transformers#migrating-from-pytorch-transformers-to-transformers

**Comment by songyouwei** [comment]
Migration done at https://github.com/songyouwei/ABSA-PyTorch/commit/b925abcdf43c51e475a64ed04984d4911b88676d
